config/config.php

```diff
@@ -5,4 +5,7 @@
 use Rector\Config\RectorConfig;
 
 return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->skip([
```

Comment:
That file is not used somewhere 🤔

Reply:
It is by the  config/6.5/... config/6.6/... files. We can import it explicitly in the main profiles directly though